AutomatedTesting/Gem/PythonTests/assetpipeline/ap_fixtures/asset_processor_fixture.py

```diff
@@ -36,5 +37,18 @@ def teardown():
         ap.stop()
 
     request.addfinalizer(teardown)
+    assert not check_ap_running("AssetProcessor.exe") and check_ap_running("AssetProcessorBatch.exe"), \
```

Comment:
Won't this only work on Windows?

Reply:
Does this need parentheses? I think right now this evaluates to (NOT AP running) AND (APB Running)